lib/loga/railtie.rb

```diff
@@ -153,26 +153,47 @@ def self.call
 
       def call
         ensure_subscriptions_attached
+        notify_for_delivered_message
         remove_log_subscriptions
       end
 
       private
 
+      def notify_for_delivered_message
+        ActiveSupport::Notifications.subscribe('deliver.action_mailer') do |*args|
+          _name, started, _finished, unique_id, data = args
+
+          mailer        = data[:mailer]
+          emails        = data[:to].join(',')
+          event_message = "Email message: #{mailer} #{unique_id} sent to: #{emails}"
+          log_message   = Loga.logger.formatter.call(6, started, nil, event_message)
+
+          Loga.logger.info(log_message)
```

Comment:
Could we take a similar approach to logging message as we do in the [Rack middleware](https://github.com/FundingCircle/loga/blob/03b2c1508962fac1d69dae9bf32fbf49c6628518/lib/loga/rack/logger.rb#L50-L72). To achieve consistency but most importantly capture metadata.

```ruby
def deliver(event)
  message = 'Sent mail to %<emails>s in %<duration>dms' % {
    emails: ...,
    duraition: ...,
  }

  loga_event = Loga::Event.new(
    data: event.slice(:mailer, :unique_id),
    message: message,
    type: 'action_mailer'
  )

  logger.info(loga_event)
end 
```

I would also be fantastic if we could listen to `recieve.action_mailer` notifications.